Apps/WebClient/src/ClientApp/src/models/userRating.ts

```diff
@@ -0,0 +1,4 @@
+export default interface UserRating {
+    // Gets or sets the rating.
+    rating: number;
```

Comment:
the rating will have to be exactly the same as the model to be serialized. I think it needs **skip** and ratingValue

Reply:
Ah gotcha. Making changes